.github/workflows/config.yml

```diff
@@ -36,26 +36,43 @@ jobs:
           export SCTPATH=HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml
           for xmlfile in workflow/sample_files/*.xml; do XMLPATH=$xmlfile python3 -c "import os; import sys; from lxml import isoschematron, etree; from pathlib import Path; sct = Path(os.environ.get('SCTPATH')).read_text(); sct_doc = etree.fromstring(bytes(sct, encoding='utf-8')); schematron = isoschematron.Schematron(sct_doc, store_report = True); xml_path = os.environ.get('XMLPATH'); xml = Path(xml_path).read_text(); xml_doc = etree.fromstring(bytes(xml, encoding='utf-8')); valid = schematron.validate(xml_doc); (not valid) and print(schematron.validation_report); (not valid) and print(f'{xml_path} fails to validate'); valid and print(f'{xml_path} validates'); (not valid) and sys.exit(1)"; done
```

Comment:
Why not separate this into container-less job and execute a python script?

Reply:
What do we gain by doing that?

Reply:
Better tracking of changes to XSD/Schematron validation, and better ci organization (these don't seem like "unit" tests)?

Reply:
I don't disagree, but I don't want to spend a bunch of time just to reorganize/rewrite this right now, it doesn't feel like a priority. If you wanted to make a new PR with that change, I'd happily accept it.